### PR TITLE
New sample - custom infowindow dialogue

### PR DESCRIPTION
**Pull request by bsnider**
This sample demonstrates the creation of a custom identity manager using the esri/request and esri/IdentityManager classes.  It does not use the API's default identity manager popup.

### CODE REVIEW

**Comment by jgravois** [comment]
it'd be better to load dojo-bootstrap via CDN than to commit a bunch of dependencies to source control.

**Comment by bsnider** [comment]
Bootstrap is loading via CDN.  It's just the Calcite Maps dependencies that are being added, because there's no CDN for it yet.

File: web-js/custom-infowindow-dialog/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#Search Widget 3D
+
+##About
+This sample shows the creation of a custom identity manager using the esri/request and esri/IdentityManager classes
+
+ Calcite Bootstrap
+
+[Live Sample](http://esri.github.io/developer-support/web-js/custom-identity-manager.html)
```
Comment (jgravois) [inline web-js/custom-infowindow-dialog/README.md]: pretty sure this should just be `index.html`

**Comment by jgravois** [comment]
> It's just the Calcite Maps dependencies that are being added

whether you're actually using it in your sample, you committed a whole directory called `./vendor/dojo-bootstrap/` which includes the raw source of  [Dojo for Bootstrap](http://xsokev.github.io/Dojo-Bootstrap/).

**Comment by jgravois** [comment]
ok, now i get it.  AL has the same stuff checked into [Calcite Maps](https://github.com/Esri/calcite-maps/tree/gh-pages/lib/js/dojo-bootstrap).

in the absence of a *real* CDN, we're still better off using something like [rawgit](https://cdn.rawgit.com/Esri/calcite-maps/master/dist/js/dojo/calcitemaps.js) than committing your sample's dependencies (and their dependencies) into version control here.

**Comment by jgravois** [comment]
@bsnider please let me know if you need help incorporating my feedback.

**Comment by bsnider** [comment]
@jgravois  I just updated the files to use rawgit.  Thanks for showing me the rawgit location for the Calcite resources.

**Comment by jgravois** [comment]
the sample looks good.  

please add a note to the README to let folks know that they'll have to supply their own webmapID (and explain how to go about finding one)

**Comment by bsnider** [comment]
@jgravois That merge was an accident... Did that throw anything off?

**Comment by jgravois** [comment]
no, there's definitely nothing wrong with merging in commits from the upstream branch you're proposing to add a new sample to.

looks good.

**Comment by bsnider** [comment]
Thanks for your help, @jgravois!

**Pull request merged**